workshops/lib/twilio-basic/signup.md

```diff
@@ -131,13 +131,9 @@ Click "I have a promo code."
 To get the actual promo code:
 
 - Ask your club leader for the Twilio promo code
-- If you are the club leader, get the promo code from
-  [`secrets.md`](https://github.com/hackclub/private/blob/master/secrets.md#twilio-promotion-code),
-  a file inside our [`private`](https://github.com/hackclub/private) GitHub
-  repository.
-- If you don't have access to that repository, please email Jonathan at
-  jonathanleung1337@gmail.com and ask for the Twilio promo code and he'll
-  quickly respond.
+- If you are the club leader, send an email to the email address
+  `jonathanleung1337@gmail.com` with your full name and school and ask for the
```

Comment:
@MaxWofford it's currently an auto-responder that sends back information so I wouldn't be able to configure my `jonathan@hackclub.io` email to do this \cc @paked